From aee623c96453d006a27676cc0f760e44db45ded8 Mon Sep 17 00:00:00 2001
From: Ian Pedowitz <ijpedowitz@google.com>
Date: Tue, 6 Feb 2018 03:07:12 +0000
Subject: [PATCH 099/306] Revert "Use the correct prebuilts for lifecycles."

This reverts commit 8c61d7b4b9dd6de37a7acc14f91f8a71623edcc1.

Reason for revert: b/72969557

Bug: 72969557
Bug: 72566647
Change-Id: I46bebf253f1dcb5abfdff2f56022675c1dd44219
(cherry picked from commit f55f5ab91419175b180fa30aeee4528ed2b3f2f9)
---
 packages/SettingsLib/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SettingsLib/common.mk b/packages/SettingsLib/common.mk
index 741db344169..3c2ca2d3188 100644
--- a/packages/SettingsLib/common.mk
+++ b/packages/SettingsLib/common.mk
@@ -16,11 +16,11 @@
 ifeq ($(LOCAL_USE_AAPT2),true)
 LOCAL_STATIC_JAVA_LIBRARIES += \
     android-support-annotations \
-    android-arch-lifecycle-common
+    apptoolkit-lifecycle-common
 
 LOCAL_STATIC_ANDROID_LIBRARIES += \
     android-support-v4 \
-    android-arch-lifecycle-runtime \
+    apptoolkit-lifecycle-runtime \
     android-support-v7-recyclerview \
     android-support-v7-preference \
     android-support-v7-appcompat \
-- 
2.17.1

